tools/scripts/helpers/issue.py

```diff
@@ -0,0 +1,162 @@
+from dataclasses import dataclass
+from typing import List
+
+from helpers.github import GitHub
+from helpers.utils import extract_url, to_datetime, extract_issue_number
+
+WEBCOMPAT_URL = "https://webcompat.com/issues/"
+
+BROWSER_MAP = {
+    "browser-fenix": "mobile",
+    "browser-firefox-mobile": "mobile",
+    "browser-focus-geckoview": "mobile",
+    "browser-android-components": "mobile",
+    "browser-firefox-ios": "mobile",
+    "browser-firefox": "desktop"
+}
+
+RESOLVED_MAP = {
+    "fixed": "site_fixed",
+    "worksforme": "site_changed",
+    "intervention": "site_changed"
```

Comment:
I don't think we should mark interventions as being resolved; we might consider an issue that's actually fixed on the site side to be lower priority than an issue where we have an intervention that requires ongoing QA+etc.

Reply:
That makes sense, I'll change that. 

I think I looked at https://github.com/webcompat/knowledge-base/blob/main/data/mouse-events-disabled.yaml#L80 and assumed we're going with `site_changed` if there is an intervention. But it appears the issue was fixed by the site. Should it be `site_fixed` for aliexpress issue then @jgraham ?